index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="../css/global.css" />
     <script src="../js/index.js" defer></script>
```

Comment:
I think one "." should be enough :)
```suggestion
    <link rel="stylesheet" href="./css/global.css" />
    <script src="./js/index.js" defer></script>
```

Reply:
I copied this way of writing from Andrea. But you're right. I should correct it 👍

Reply:
I've decided to write without the dots.